### DISCUSSION TITLE
Release 9.0.0 - CacheDirectory property missing

### DISCUSSION THREAD

**Discussion by MattiaCamponesco** [dlemstra/Magick.NET#1120] [General]
Dear all, updating my packages to the latest version 9.0.0 (more in deep, Magick.NET.Core and Magick.NET-Q8-AnyCPU) I found out that there's no more support to the cache directory property and I didn't find any other class in which could it be migrated. 

![image](https://user-images.githubusercontent.com/74658001/150738635-10f10293-67d3-442c-af45-0ab2cc5b51a0.png)

**Comment by dlemstra** [comment]
Which version of .NET are you using? This was removed in version 9.0.0 because support for .NET 2.0 was removed from the AnyCPU build. This should not be required when you use a .NET version that support .NET Standard 2.0. If you are on an older version you could switch to the x86 or x64 instead.

**Reply by MattiaCamponesco** [reply]
Ok, you already answered me, I think it was a refuse form previous version.
Now we are currently running on .NET Framework 4.7.2 so I can remove this feature.
